e2e/e2e_test.go

```diff
@@ -87,11 +87,12 @@ func TestGRPCIntegration(t *testing.T) {
 	}
 	defer serverCloser()
 
-	agentCloser, err := StartPortForward(ctx, cfg, "https", parcaAgent, ns, "7071")
-	if err != nil {
-		require.NoError(t, err, "failed to start port forwarding Parca Agent: %v", err)
-	}
-	defer agentCloser()
+	// // Port 7071 may already be in use by docker-proxy on the host (e.g. Minikube with none driver)
```

Comment:
If we didn't this won't be used, let's delete the code.

Reply:
It was not straightforward to troubleshoot, the comment might save someone's time in a not so far future?

Reply:
We can leave the comment. Do we need to code?